coreneuron/nrniv/main1.cpp

```diff
@@ -399,9 +399,8 @@ extern "C" int run_solve_core(int argc, char** argv) {
     }
 
     // initializationa and loading functions moved to separate
-    Instrumentor::phase_begin("load-model");
+    Instrumentor::phase p("load-model");
     nrn_init_and_load_data(argc, argv, !configs.empty());
-    Instrumentor::phase_end("load-model");
 
```

Comment:
`load-model` should only measure `nrn_init_and_load_data`. If I am not mistaken, `p` will measure more than this here (whole block scope). Need to introduce a local block? 

```
{
    Instrumentor::phase p("load-model");
    nrn_init_and_load_data(argc, argv, !configs.empty());
}
```